src/registrar/fixtures/fixtures_suborganizations.py

```diff
@@ -0,0 +1,87 @@
+import logging
+from faker import Faker
+from django.db import transaction
+
+from registrar.models.portfolio import Portfolio
+from registrar.models.suborganization import Suborganization
+
+
+fake = Faker()
+logger = logging.getLogger(__name__)
+
+
+class SuborganizationFixture:
+    """
+    Creates 2 pre-defined suborg with the infrastructure to add more.
+
+    Depends on fixtures_portfolios.
+
+    Make sure this class' `load` method is called from `handle`
+    in management/commands/load.py, then use `./manage.py load`
+    to run this code.
+    """
+
+    SUBORGS = [
+        {
+            "name": "Take it Easy",
+        },
+        {
+            "name": "Welcome to the Machine",
+        },
+    ]
+
+    @classmethod
+    def load(cls):
+        """Creates suborganizations."""
+        logger.info(f"Going to load {len(cls.SUBORGS)} suborgs")
+
+        with transaction.atomic():
+            portfolios = cls._get_portfolios()
+            if not portfolios:
+                return
+
+            suborgs_to_create = cls._prepare_suborgs_to_create(portfolios)
+            cls._bulk_create_suborgs(suborgs_to_create)
+
+    @classmethod
+    def _get_portfolios(cls):
```

Comment:
I disagree. There's no benefit in adding multiple portfolios WITH multiple nested suborgs each, because unlike requests, we do not have multiple variations (such as statuses) to test. If we need to add more portfolios, we can easily do so and they would have no suborgs, which is fine for fixtures.

If we need to add multiple suborgs to a portfolio, we can easily do so on portfolio1 or 2 and that would be sufficient.

Reply:
You can add more portfolios, the suborg fixtures will ignore them